system_baseline/views/v0.py

```diff
@@ -258,6 +260,32 @@ def create_baseline(system_baseline_in):
     return baseline.to_json()
 
 
+@metrics.baseline_create_requests.time()
+@metrics.api_exceptions.count_exceptions()
+def copy_baseline_by_id(baseline_ids, display_name):
```

Comment:
I was thinking that if the expectation is a single baseline ID, that treating this as an array of baseline IDS seemed odd from a developer understanding the code.

Reply:
I wanted the openapi definitions to all live in the same section "/baselines/{baseline_ids}". However doing that requires me to do weird stuff like this.

I'll update and put the verbs for single baselines in its own section.